sites/orgdocs/docs/designs/sabrina/options/backdartangle/readme.mdx

```diff
@@ -0,0 +1,8 @@
+---
+title: 'Back dart angle'
+sidebar_class_name: hidden
+---
+
+Sabrina creates a virtual dart to prevent sticking up fabric in the center of the back neck.
```

Comment:
There is kinda a dart in the neck back that pulls together the straps, but instead of sewing the actual dart, the straps are just rotated towards the midline.

Reply:
Maybe you have some suggestion to express that better.